Dockerfile.msb

```diff
@@ -1,16 +1,23 @@
 # MULTI-STAGE BUILD for IBM Spectrum Scale CSI Driver
 
 FROM golang:1.13.1 AS builder
-RUN wget -q -O $GOPATH/bin/dep https://github.com/golang/dep/releases/download/v0.5.1/dep-linux-amd64
+RUN wget -q -O $GOPATH/bin/dep https://github.com/golang/dep/releases/download/v0.5.1/dep-linux-amd64 
```

Comment:
I see extra space at the end of this line. Apart for this changed looks good to me

Reply:
Removed the white space.